projects/plugins/jetpack/modules/sso/jetpack-sso-admin-create-user.css

```diff
@@ -14,8 +14,12 @@
 	width: 25em;
 }
 
+#createuser .form-field [type=checkbox] {
+	width: auto;
```

Comment:
The checkbox is expanding a bit when its checked.

https://github.com/Automattic/jetpack/assets/10482592/348a5344-c57e-4a31-b399-9c734382e6af


Lets update this to `width: 1rem`.